sceptre/resolvers/__init__.py

```diff
@@ -127,23 +251,6 @@ def _no_recursive_get(self, stack: "stack.Stack"):
         finally:
             setattr(stack, get_status_name, False)
 
-    def get_setup_resolver_for_stack(
-        self, stack: "stack.Stack", resolver: Resolver
-    ) -> Resolver:
-        """Obtains a clone of the resolver with the stack set on it and the setup method having
-        been called on it.
-
-        :param stack: The stack to set on the Resolver
-        :param resolver: The Resolver to clone and set up
-        :return: The cloned resolver.
-        """
-        # We clone the resolver when we assign the value so that every stack gets its own resolver
-        # rather than potentially having one resolver instance shared in memory across multiple
-        # stacks.
-        clone = resolver.clone(stack)
-        clone.setup()
-        return clone
-
```

Comment:
This method is no longer necessary. It's all been moved into the CustomYamlTagBase's `clone_for_stack` method.